kmp-nativecoroutines-core/src/appleTest/kotlin/com/rickclephas/kmp/nativecoroutines/NativeFlowTests.kt

```diff
@@ -19,7 +21,7 @@ class NativeFlowTests {
     }
 
     @Test
-    fun `ensure completion callback is invoked`() = runBlocking {
+    fun `ensure completion callback is invoked`() = runTest {
         val flow = flow<RandomValue> {  }
         val job = Job()
         val nativeFlow = flow.asNativeFlow(CoroutineScope(job))
```

Comment:
I think we need to change this to the `TestScope` from `runTest` but let's see what happens with https://github.com/Kotlin/kotlinx.coroutines/issues/3195